### PR TITLE
"exit group" verification popup

### PR DESCRIPTION
**Pull request by Chen-Ben-yashar**
upon "Exit Group" button pressed - a popup will show and verify that this is what the user wants

bug: if the "reset choices" button is pressed earlier - the icons look funny

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -6,9 +6,19 @@
     <string name="dayRepresentation">Day Representation</string>
     <string name="timeInDayRepresentation">Time In Day Representation</string>
 
+    <string name="EXIT_GROUP_TITLE">Exit Group</string>
+    <string name="EXIT_GROUP_MESSAGE">Are you sure You want To Exit Group?</string>
```
Comment (sapirshp) [inline app/src/main/res/values/strings.xml]: should be "Are You Sure You Want To Exit Group?"

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -6,9 +6,19 @@
...
+    <string name="EXIT_ANSWER">Exit</string>
+    <string name="DONT_EXIT_ANSWER">No, Stay</string>
 
 
 
+    <!--private String EXIT_GROUP_TITLE = "Exit Group";-->
+    <!--private String EXIT_GROUP_MESSAGE = "Are you sure You want To Exit Group?";-->
+    <!--private String EXIT_ANSWER = "Exit";-->
+    <!--private String DONT_EXIT_ANSWER = "No, Stay";-->
```
Comment (sapirshp) [inline app/src/main/res/values/strings.xml]: delete the comments here

File: app/src/main/java/com/example/meetapp/MenuHandler.java [hunk 3]
```diff
@@ -26,7 +28,7 @@
     ArrayList<TimeSlot> slotsToReset = new ArrayList<>();
 
 
-    MenuHandler(Dialog dialog, int memberNum, String membersNames){
+     MenuHandler(Dialog dialog, int memberNum, String membersNames){
         this.addMemberDialog = dialog;
         membersAmount = memberNum;
```
Comment (orenpinto) [inline app/src/main/java/com/example/meetapp/MenuHandler.java]: add this before membersAmount

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Chen-Ben-yashar) [reply app/src/main/res/values/strings.xml]: fixed

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (Chen-Ben-yashar) [reply app/src/main/res/values/strings.xml]: fixed

File: app/src/main/java/com/example/meetapp/MenuHandler.java (hunk 3 above)
Comment (Chen-Ben-yashar) [reply app/src/main/java/com/example/meetapp/MenuHandler.java]: not mine, but fixed :)

**Pull request merged**